sycl/test/usm/mixed.cpp

```diff
@@ -1,5 +1,5 @@
-// RUN: %clang -std=c++11 -fsycl %s -o %t1.out -lstdc++ -lOpenCL -lsycl -DINTEL_USM
-// RUN: %CPU_RUN_PLACEHOLDER %t1.out
+// RUN: %clang -std=c++11 -fsycl %s -o %t1.out -lstdc++ -lOpenCL -lsycl
+// -DINTEL_USM RUN: %CPU_RUN_PLACEHOLDER %t1.out
```

Comment:
`-DINTEL_USM` should be one line above. Right?

Reply:
That was a relic.  Removed.